### ISSUE TITLE
Comments

### ISSUE THREAD

**Issue by andersontr15** [jesushilariohernandez/weather-app#1]
@jesushilariohernandez 

I apologize for never getting back to you about the project. I've looked at it briefly and it looks good. I see you're using the `XMLHttpRequest` API  instead of `fetch`. Any reason for that? 

Cheers!

**Comment by jesushilarioh** [comment]
@andersontr15  

Hey, thanks for getting back...! I'm not familiar with `fetch`. This is actually the first time I'm hearing about it... I thought the `XMLHttpRequest` was the only way to implement an API in javascript. I'm really new to programming... so, forgive my naivety. :) I'm looking at fetch now, there are a few keywords I don't know... *haaa...* more research... lol!

**Comment by andersontr15** [comment]
@jesushilariohernandez  Yeah no problem. I didn't see the message for awhile and then just got busy.  `fetch` is just a browser API that makes doing http requests a little easier with the API it exposes, as well as being wrapped in a promise so it has the `.then()` method. 

No man, this field is vastly overwhelming. I have 1 year of experience at my job now and I feel like I've barely scratched the surface of what I know..and that's mainly just using javascript.
